CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v3.2.2
+* Fixed random failures during starting operation caused by wrong CSRF token handling
```

Comment:
Can we put here what was the issue, e.g. some 403 returned from backend

Reply:
Cannot find a failed job with this error...